components/logins/ios/Logins/LoginsStorage.swift

```diff
@@ -372,6 +372,20 @@ open class LoginsStorage {
         }
     }
 
+    /// Get the set of potential duplicates ignoring the username of `login`.
+    open func potentialDupesIgnoringUsername(to login: LoginRecord) throws -> [LoginRecord] {
+        let json = try login.toJSON()
+        return try queue.sync {
+            let engine = try self.getUnlocked()
+            let buffer = try LoginsStoreError.unwrap { err in
+                sync15_passwords_potential_dupes_ignoring_username(engine, json, err)
```

Comment:
This needs to be updated to protobuf input also?

Reply:
Forgot to save before committing 😅